edx_rest_api_client/client.py

```diff
@@ -1,13 +1,24 @@
 import datetime
+import os
 
 import requests
+import requests.utils
 import slumber
 
 from edx_rest_api_client.auth import BearerAuth, JwtAuth, SuppliedJwtAuth
+from edx_rest_api_client.__version__ import __version__
 
 
 class EdxRestApiClient(slumber.API):
 
+    @classmethod
+    def user_agent(cls):
+        return "{} edx-rest-api-client/{} {}".format(
+            requests.utils.default_user_agent(),  # e.g. "requests/1.7.1"
+            __version__,  # version of this client
+            os.environ.get("EDX_REST_API_CLIENT_NAME", "unknown")
```

Comment:
Would "unknown_client_name" help some better determine how to set this if it is unset?

Reply:
Sure, I'll make that change.